### PR TITLE
Link to API Doc in README

### PR DESCRIPTION
**Pull request by jessitron**
triggered by: I wonder whether npm install works for our build because it doesn't locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 [Atomist][atomist] software delivery machine (SDM) extension Pack for
 an Atomist SDM to integrate [Kubernetes][kubernetes] deployments.
 
+[API Doc][apidoc]
```
Comment (ddgenome) [inline README.md]: Should this be in a sentence or something?

File: README.md [hunk 2]
```diff
@@ -44,9 +46,13 @@ behavior to code-of-conduct@atomist.com.
...
 
 Please see [docs.atomist.com][atomist-doc] for
-[developer][atomist-doc-sdm] documentation.
+[developer][atomist-doc-sdm] documentation and for high-level documentation of [this pack][atomist-doc-k8-pack].
+
+Also see the TypeScript [API Doc][apidoc] for this pack.
 
 [atomist-doc-sdm]: https://docs.atomist.com/developer/sdm/ (Atomist Documentation - SDM Developer)
+[atomist-doc-k8-pack]: https://docs.atomist.com/pack/k8s/ (Atomist K8s Pack Documentation)
+[apidoc]: https://atomist.github.io/sdm-pack-k8s
```
Comment (ddgenome) [inline README.md]: This link should not have an "s" at the end, right?

File: README.md [hunk 3]
```diff
@@ -44,9 +46,13 @@ behavior to code-of-conduct@atomist.com.
 ## Documentation
 
 Please see [docs.atomist.com][atomist-doc] for
-[developer][atomist-doc-sdm] documentation.
+[developer][atomist-doc-sdm] documentation and for high-level documentation of [this pack][atomist-doc-k8-pack].
+
+Also see the TypeScript [API Doc][apidoc] for this pack.
```
Comment (ddgenome) [inline README.md]: Is the above link necessary if this one exists?

File: README.md [hunk 4]
```diff
@@ -44,9 +46,13 @@ behavior to code-of-conduct@atomist.com.
...
 
 [atomist-doc-sdm]: https://docs.atomist.com/developer/sdm/ (Atomist Documentation - SDM Developer)
+[atomist-doc-k8-pack]: https://docs.atomist.com/pack/k8s/ (Atomist K8s Pack Documentation)
```
Comment (ddgenome) [inline README.md]: This link seems to be https://docs.atomist.com/pack/kubernetes/

File: README.md (hunk 3 above)
Comment (jessitron) [reply README.md]: I wanted one at the beginning

File: README.md (hunk 4 above)
Comment (jessitron) [reply README.md]: thanks! I was going to check that. Main motivation for the PR was to check the build.

**Pull request merged**